### ISSUE TITLE
Can't Save and load subsets

### ISSUE THREAD

**Issue by matix931** [Chicken-Bones/NotEnoughItems#87]
Clicking with right mouse button on "NEI Subsets" button not working.

Forge 1208.
CCC 1.0.3.26
NEI 1.0.3.56

**Comment by dredhorse** [comment]
I have the same issue, right clicking on NEI Subsets don't work.

MC 1.7.10
Forge 1291
CCC 1.0.4.35
NEI 1.0.4.90

**Comment by Chicken-Bones** [comment]
Feature was never reimplemented. May return at some point in the future

**Comment by dredhorse** [comment]
Is there a way to copy the config in some way?

**Comment by jadenPete** [comment]
Yes, I Would really like this feature. I have many mods that add blocks and items with missing textures. So then I have like 50 blocks with pink and black checkerboard textures. It would be nice if I could just save it to a list and load it everytime I create a world.
